urifind.go

```diff
@@ -5,17 +5,17 @@ import (
 	"bytes"
 	_ "crypto/sha512"
 	"errors"
-	"golang.org/x/net/html"
-	"golang.org/x/net/html/atom"
-	"golang.org/x/net/html/charset"
-	"golang.org/x/text/transform"
 	"io"
 	"log"
 	"net/http"
 	"regexp"
 	"strconv"
 	"strings"
 	"time"
+
+	"golang.org/x/net/html"
```

Comment:
Ja: man gruppiert grundsätzlich gleichartige imports. In diesem Fall sind das standard library imports und third-party imports. Andere Arten wären umbenannte imports (`import "mathrand" "math/rand"`) und imports für side effects (`import _ "image/jpeg"`).